header-rewrite-filter/utility.cc

```diff
@@ -46,6 +46,27 @@ BooleanOperatorType StringToBooleanOperatorType(absl::string_view bool_operator)
     }
 }
 
+FunctionType StringToFunctionType(absl::string_view function) {
+    if (function.compare(DYNAMIC_VALUE_HDR) == 0) {
+        return FunctionType::GetHdr;
+    } else if (function.compare(DYNAMIC_VALUE_URL_PARAM) == 0) {
+        return FunctionType::Urlp;
+    } else {
+        return FunctionType::InvalidFunctionType;
+    }
+}
+
+absl::Status GetFunctionArgument(FunctionType function_type, std::string& function_expression){
+    try {
+        auto start = function_expression.find_first_of("(");
+        auto end = function_expression.find_last_of(")");
```

Comment:
Rm? Looks to be unused

Reply:
Since some functions might not take an argument, I was originally thinking of returning a value based on the function type but it's simpler if this function doesn't have to deal with that logic -- each respective dynamic function can focus on that.